docs/function.md

```diff
@@ -110,231 +108,185 @@ let eleven = addFive(6);
 let twelve = addFive(7);
 ```
 
-Actually, the above `add` is nothing but syntactic sugar for this:
+If you want to partially call a function with an argument that is not the first
+argument use a `_` to skip arguments:
 
 ```reason
-let add = (x) => (y) => x + y;
```

Comment:
This was actually an interesting bit. I'll think about including it in future iterations somehow.